src/issuance.rs

```diff
@@ -653,6 +671,54 @@ mod tests {
         )
     }
 
+    fn generate_identity_point_asset_base() -> AssetBase {
```

Comment:
lets call it `identity_point()`. Also add a proper comment to the function.

Reply:
Done.